Submitted By: Ryan Oliver <ryan(dot)oliver(at)pha(dot)com(dot)au>
Date: 2005-10-20
Initial Package Version: 5.8.7
Origin: Ryan Oliver
Description: this patch allows perl to be installed in /usr/lib32 or /usr/lib64 instead of /usr/lib.

--- perl-5.8.7/Configure-ORIG	2005-10-20 11:49:47.571389008 +1000
+++ perl-5.8.7/Configure	2005-10-20 12:30:35.571236464 +1000
@@ -5930,6 +5930,8 @@
 : The default "style" setting is made in installstyle.U
 case "$installstyle" in
 *lib/perl5*) set dflt privlib lib/$package/$version ;;
+*lib32/perl5*) set dflt privlib lib32/$package/$version ;;
+*lib64/perl5*) set dflt privlib lib64/$package/$version ;;
 *)	 set dflt privlib lib/$version ;;
 esac
 eval $prefixit
@@ -6433,6 +6435,8 @@
 case "$sitelib" in
 '') case "$installstyle" in
 	*lib/perl5*) dflt=$siteprefix/lib/$package/site_$prog/$version ;;
+	*lib32/perl5*) dflt=$siteprefix/lib32/$package/site_$prog/$version ;;
+	*lib64/perl5*) dflt=$siteprefix/lib64/$package/site_$prog/$version ;;
 	*)	 dflt=$siteprefix/lib/site_$prog/$version ;;
 	esac
 	;;
@@ -6560,6 +6564,8 @@
 		prog=`echo $package | $sed 's/-*[0-9.]*$//'`
 		case "$installstyle" in
 		*lib/perl5*) dflt=$vendorprefix/lib/$package/vendor_$prog/$version ;;
+		*lib32/perl5*) dflt=$vendorprefix/lib32/$package/vendor_$prog/$version ;;
+		*lib64/perl5*) dflt=$vendorprefix/lib64/$package/vendor_$prog/$version ;;
 		*)	     dflt=$vendorprefix/lib/vendor_$prog/$version ;;
 		esac
 		;;
